vendor/k8s.io/klog/v2/k8s_references.go

```diff
@@ -77,6 +77,8 @@ func KRef(namespace, name string) ObjectRef {
 }
 
 // KObjs returns slice of ObjectRef from an slice of ObjectMeta
+//
+// DEPRECATED: Use KObjSlice instead, it has better performance.
```

Comment:
is there an issue tracking switching use to this in a follow up?

Reply:
There is now: https://github.com/kubernetes/kubernetes/issues/110737